v3/pkg/application/application.go

```diff
@@ -1046,8 +1047,16 @@ func (a *App) Paths(selector Paths) []string {
 	return pathdirs[selector]
 }
 
+// OpenDirectory opens the system file explorer to the specified directory.
 func (a *App) OpenDirectory(path string) error {
```

Comment:
Do we want to leave both `OpenDirectory` and `OpenFileManager` methods in the runtime API?

Technically, `OpenDirectory` also opens the file manager but without the possibility to select a file

Reply:
Yes, let's remove it!